iamproperty/iamproperty.github.io
# Ds 45/navbar

## Summary of Changes:
1. Removed unnecessary JS modules from the main.js file
2. Updated CSS for nav component
3. Vue JS nav component
4. Unit tests and visual tests setup
5. Documentation for nav component
6. Create new standalone example for iamsold admin page and updated the iamsold homepage example

## Rationale
A consistent nav component that can be used on external facing sites as well as admin sections.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /components/nav

## Ticket(s)
DS-45

## Checklist

## Notes
I have a second task (DS-46) to create the more complex parts of the navigation. This includes:

- Drop down sub links (Marketplace admin section uses these)
- Mega menu (iamsold and iamproperty external sites use these)
- Search bar drop down (new iamsold external site design uses)
- Search bar inline (iamsold admin site uses)
